html-relative-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Contact Us</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
+    <li><a href="../projects/projects.html">Projects</a></li>
+    <li><a href="../contact-us/contact-us.html">Contact Us</a></li>
```

Comment:
With this relative path, you are exiting the current directory just to go right back into it. Since you are already in the directory for the file you are linking, you can just state the name of the file this is linking to.